src/index.html

```diff
@@ -23,6 +23,8 @@
     <meta name="twitter:description" content="Women Hack For Non-Profits | Home" />
     <meta name="twitter:image" content="img/whfnpLogoSocialMediaIcon.png" />
     <!--Styles-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```

Comment:
- i would improve this by using [bootsrap sass](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss) and only importing what is needed into the current sass example here: https://github.com/womenhackfornonprofits/fashion-revolution-django/tree/master/static-src/styles

The site is not performing great as it is this is extra resources to load.